PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,19 @@
 <!--
 Thanks for contributing a pull request! Please ensure you have taken a look at
-the contribution guidelines: https://github.com/giotto-ai/giotto-tda/blob/master/CONTRIBUTING.md#pull-request-checklist
+the contribution guidelines: https://giotto-ai.github.io/gtda-docs/dev/contributing/#guidelines
```

Comment:
amazing, slowly migrating. Do you want to point to dev, or maybe nightly? I think we discussed that the 'default' version would be nightly, but maybe for contributors, you want to point them to dev?

Reply:
Yes, I'd think dev is the more relevant one for contributors.